testing/integration/examples/integration.rs

```diff
@@ -0,0 +1,73 @@
+use fvm::executor::{ApplyKind, Executor};
+use fvm_integration_tests::tester::Tester;
+use fvm_ipld_encoding::tuple::*;
+use fvm_shared::address::Address;
+use fvm_shared::bigint::BigInt;
+use fvm_shared::message::Message;
+use fvm_shared::state::StateTreeVersion;
+use fvm_shared::version::NetworkVersion;
+use num_traits::Zero;
+use wabt::wat2wasm;
+
+const WAST: &str = r#"
+;; Mock invoke function
+(module
+  (func (export "invoke") (param $x i32) (result i32)
+    (i32.const 1)
+  )
+)
+"#;
+
+#[derive(Serialize_tuple, Deserialize_tuple, Clone, Debug)]
+struct State {
+    empty: bool,
+}
+
+pub fn main() {
```

Comment:
Would it be possible to have these as Rust tests?

Reply:
I used the `hello_world` example that you created for the FVM EBP 👍